Tribler/Core/Libtorrent/LibtorrentMgr.py

```diff
@@ -129,31 +128,30 @@ def shutdown(self):
     def create_session(self, hops=0, store_listen_port=True):
         settings = {}
 
+        is_1_1_version = LooseVersion(self.get_libtorrent_version()) >= LooseVersion("1.1.0")
+
         # Due to a bug in Libtorrent 0.16.18, the outgoing_port and num_outgoing_ports value should be set in
         # the settings dictionary
         settings['outgoing_port'] = 0
         settings['num_outgoing_ports'] = 1
 
+        if is_1_1_version:
+            settings["dht_bootstrap_nodes"] = ""
```

Comment:
I imagine you would want to do something like: ``','.join(DEFAULT_DHT_ROUTERS)``